src/main/msp/msp.c

```diff
@@ -2196,6 +2197,9 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, uint8_t cmdMSP,
             // Added in API version 1.43
             gpsRescueConfigMutable()->ascendRate = sbufReadU16(src);
             gpsRescueConfigMutable()->descendRate = sbufReadU16(src);
+        if (sbufBytesRemaining(src) >= 1) {
```

Comment:
This isn't correct - you are duplicating the check from https://github.com/betaflight/betaflight/pull/9026/files#diff-eefd576c86ccfa8b6cdf98081806a7a8R2196.